draft-kazuho-httpbis-priority.md

```diff
@@ -257,27 +322,24 @@ end-to-end rather than hop-by-hop.
 It should also be noted that the use of a header field carrying a textual value
 makes the prioritization scheme extensible; see the discussion below.
 
-## Why are there Only Three Levels of Urgency?
+## Why do Urgencies Have Meanings?
 
 One of the aims of this specification is to define a mechanism for merging
 client- and server-provided hints for prioritizing the responses.  For that to
 work, each urgency level needs to have a well-defined meaning.  As an example, a
 server can assign the highest precedence among the non-blocking responses to an
-HTTP response carrying an icon, because the meaning of "non-blocking" is
-shared among the endpoints.
+HTTP response carrying an icon, because the meaning of `urgency=1` is shared
+among the endpoints.
 
-This specification restricts itself to defining just three levels of urgency, in
-order to provide sufficient granularity for prioritizing responses for ordinary
-web browsing, at minimal complexity.
+This specification restricts itself to defining a minimum set of urgency levels
+in order to provide sufficient granularity for prioritizing responses for
+ordinary web browsing, at minimal complexity.
 
 However, that does not mean that the prioritization scheme would forever be
 stuck to the three levels.  The design provides extensibility.  If deemed
-necessary, it would be possible to divide any of the three urgency levels into
-sub-levels by defining a new parameter.  As an example, a server could assign
-an `importance` parameter to the priority of each image that it provides, so
-that an intermediary could prioritize certain images above others.  Or, a
-graphical user-agent could send a `visible` parameter to indicate if the
-resource being requested is within the viewport.
+necessary, it would be possible to subdivide any of the three urgency levels
```

Comment:
there are more than three urgency levels now

Reply:
Nice catch! Fixed in f98aa72.